.github/workflows/cit-workflow.yml

```diff
@@ -0,0 +1,41 @@
+name: Combined CIT Workflow
+
+on:
+  push:
+    branches: [main]
+
+  pull_request:
+    branches: [main]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ (github.event.pull_request && github.event.pull_request.number) || github.ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  badger:
+    uses: ./.github/actions/e2e/ci-e2e-badger.yaml@main
```

Comment:
try creating a dummy workflow first and referencing it like this

Reply:
i tried but the same error. It seems GitHub Actions expects workflows to be in the .github/workflows directory. How should I proceed with this now. Is the approach to add all the workflows as jobs in a single file ok or is there any other solution to it?

Reply:
the fact that it gives a specific error makes me think that the workflow inclusion is actually supported, that is why I am suggesting:
  * try creating two dummy workflows A and B in the workflow dir
  * make A include and run B
  * check if the runner executes A successfully (with B), but standalone B is not executed

Reply:
And I would prefer you do these experiments in your own test repo, there is no need to run a battery of expensive jaeger tests while you don't have a solve for the workflows

Reply:
If including workflow in a workflow does not work, then the alternative is to define the jobs in a combined workflow and use the existing files as actions.

Reply:
I tested this in my test repo and this approach works perfectly fine I just needed to add workflow_call on the cit workflows.